packages/component/src/Styles/defaultStyleOptions.js

```diff
@@ -111,9 +111,13 @@ const DEFAULT_OPTIONS = {
   suggestedActionDisabledTextColor: undefined, // defaults to subtle
   suggestedActionHeight: 40,
   suggestedActionImageHeight: 20,
-  suggestedActionLayout: 'carousel', // either "carousel" or "stacked"
+  suggestedActionLayout: 'carousel', // either 'carousel' or 'stacked'
   suggestedActionTextColor: null,
 
+  // Suggeseted actions 'stacked' layout
+  suggestedActionsStackedHeight: undefined, // defaults to 'auto'
+  suggestedActionsStackedOverflow: undefined,
```

Comment:
I think since `suggestedActionsStackedOverflow` doesn't default to a value we want to have it set to null. Not sure exactly why we're using that convention, but we are. @compulim Thoughts?

Reply:
Ah I had forgotten, thanks